appeals/web/src/server/appeals/appeal-details/site-ownership/__tests__/site-ownership.test.js

```diff
@@ -0,0 +1,70 @@
+import { parseHtml } from '@pins/platform';
+import supertest from 'supertest';
+import { appealData, appellantCaseDataNotValidated } from '#testing/app/fixtures/referencedata.js';
+import { createTestEnvironment } from '#testing/index.js';
+import nock from 'nock';
+
+const { app, installMockApi, teardown } = createTestEnvironment();
+const request = supertest(app);
+const appealId = appealData.appealId;
+const appellantCaseId = appealData.appellantCaseId;
+const baseUrl = `/appeals-service/appeal-details/${appealId}/appellant-case`;
+describe('site-ownership', () => {
+	beforeEach(installMockApi), afterEach(teardown);
+
+	describe('GET /change', () => {
+		it('should render changeSiteOwnership page', async () => {
+			nock('http://test/')
+				.get(`/appeals/${appealId}/appellant-cases/${appellantCaseId}`)
+				.reply(200, {
+					...appellantCaseDataNotValidated
```

Comment:
you should be able to just return `appellantCaseDataNotValidated` here rather than spreading the object